src/app/organisations/getUsersByRoles.js

```diff
@@ -0,0 +1,55 @@
+const logger = require('./../../infrastructure/logger');
+const {getOrganisationByTypeAndIdentifier} = require("../../infrastructure/organisations");
+const { getServiceUsers } = require("../../infrastructure/access");
+const { usersByIds } = require('../../infrastructure/directories');
+
+const getUsersByRoles = async (req, res) => {
+    const { correlationId, clientCorrelationId } = req;
+    const roles = req.query.roles?req.query.roles.split(','):null;
+
+    try {
+        logger.info(`Getting users for UKPRN ${req.params.id} (correlationId: ${correlationId}, client correlationId: ${clientCorrelationId}`, {
+            correlationId,
+            clientCorrelationId
+        });
+    if (!req.client.id || !req.params.id || !roles) {
```

Comment:
Why don't we just return all the users that have access to the service for the specified org instead of returning 404 if no roles are specified